frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingDialogs/createDrawingDialog.component.tsx

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
I know this is also how containers works, so this is potentially something for a separate issue later, but I feel like some of this should surely be reusable with the edit modal. Do you know if there's a reason we're not reusing components between them?